runtime/src/asset.rs

```diff
@@ -342,6 +343,7 @@ decl_module! {
             total_supply: T::Balance,
             divisible: bool,
             asset_type: AssetType,
+            funding_round: Option<Vec<u8>>,
```

Comment:
Super minor, but I have a small preference for optional arguments coming at the end of the parameter list.